biogtr/models/attention_head.py

```diff
@@ -0,0 +1,105 @@
+"""Module containing different components of multi-head attention heads."""
+
+import torch
+import torch.nn.functional as F
+
+# todo: add named tensors
+
+
+class MLP(torch.nn.Module):
```

Comment:
These classes are also adapted from gtr repo so we should just make sure to cite.